### PR TITLE
10.0 wip

### PR DESCRIPTION
**Pull request by gwolf2u**
new stuff

### CODE REVIEW

File: core/res/res/values/cr_config.xml [hunk 1]
```diff
@@ -19,6 +19,9 @@
     <!-- Whether device has dash charging support -->
     <bool name="config_hasDashCharger">false</bool>
 
+    <!-- Whether device has warp charging support -->
```
Comment (neobuddy89) [inline core/res/res/values/cr_config.xml]: XML entries should be in end. Not in middle.

**Review by neobuddy89 — CHANGES_REQUESTED** [review]
Better add options instead xml overlay.
https://github.com/Project-Xtended/frameworks_base/commit/8351ba85af149361d54a70a5d2a5d5a75e494440

**Review by neobuddy89 — COMMENTED** [review]
Why do you want old style icons?

File: core/java/android/provider/Settings.java [hunk 2]
```diff
@@ -3160,6 +3160,12 @@ public boolean validate(String value) {
          */
         public static final int END_BUTTON_BEHAVIOR_DEFAULT = END_BUTTON_BEHAVIOR_SLEEP;
 
+        /**
+        * List of Apps hidden from recents
+        * @hide
+        */
+        public static final String HIDE_FROM_RECENTS_LIST = "hide_from_recents_list";
```
Comment (neobuddy89) [inline core/java/android/provider/Settings.java]: Why is this placed here? Should be on end where last setting was added by us.

File: core/res/res/values/cr_config.xml (hunk 1 above)
Comment (gwolf2u) [reply core/res/res/values/cr_config.xml]: wanted to keep it near the one with Dash charge
doesn't matter that much anyway
want it on the end?

**Comment by gwolf2u** [comment]
> Better add options instead xml overlay.
> [Project-Xtended/frameworks_base@8351ba8](https://github.com/Project-Xtended/frameworks_base/commit/8351ba85af149361d54a70a5d2a5d5a75e494440)

don't know if better to have option, yet will have a look

**Comment by gwolf2u** [comment]
> Why do you want old style icons?

preserver status bar area on notch devices (including poco)

File: core/java/android/provider/Settings.java (hunk 2 above)
Comment (gwolf2u) [reply core/java/android/provider/Settings.java]: can add at end no prob

File: core/res/res/values/cr_config.xml (hunk 1 above)
Comment (neobuddy89) [reply core/res/res/values/cr_config.xml]: okay.. can add. will add properly in next rebase.

**Comment by neobuddy89** [comment]
make sure you have follow up commits too like
https://github.com/omnirom/android_frameworks_base/commit/b8054b94caefd8be007db2352efdee9f72ca1561

**Review by neobuddy89 — COMMENTED** [review]
It doesn't make sense to match pulse light color with wallpaper color as pulse color doesn't appear on wallpaper.

**Comment by gwolf2u** [comment]
> It doesn't make sense to match pulse light color with wallpaper color as pulse color doesn't appear on wallpaper.

whoops
merged that by mistake

**Pull request closed without merging**